docs/gathered-data.md

```diff
@@ -13,1040 +13,1878 @@ archives are packaged as `.tar.gz` files in `/var/lib/insights-operator`.
+- Response is an array of arrays of namespaces with overlapping UIDs. Each namespace is represented by its name
+and the UID range value from the "openshift.io/sa.scc.uid-range" annotation
+
+### Sample data
+- [docs/insights-archive-sample/config/namespaces_with_overlapping_uids.json](./insights-archive-sample/config/namespaces_with_overlapping_uids.json)
+
+### Location in archive
+- `config/namespaces_with_overlapping_uids.json`
+
+### Config ID
+`clusterconfig/overlapping_namespace_uids`
 
-The Kubernetes api https://github.com/kubernetes/client-go/blob/master/kubernetes/typed/core/v1/namespace.go
-Response is an array of arrays of namespaces with overlapping UIDs. Each namespace is represented by its name and the UID range value
-from the "openshift.io/sa.scc.uid-range" annotation
+### Released version
+- 4.11.0
 
-* Location in archive: config/namespaces_with_overlapping_uids
-* Id in config: clusterconfig/overlapping_namespace_uids
-* Since versions:
-  - 4.8.41
-  - 4.9.31
-  - 4.10.12
-  - 4.11+
+### Backported versions
+- 4.8.41+
+- 4.9.31+
+- 4.10.12+
+
+### Changes
+None
 
 
 ## NetNamespace
 
-collects NetNamespaces networking information
+Collects NetNamespaces networking information.
+
+### API Reference
+- https://github.com/openshift/client-go/blob/master/network/clientset/versioned/typed/network/v1/netnamespace.go
+
+### Sample data
+- [docs/insights-archive-sample/config/netnamespaces.json](./insights-archive-sample/config/netnamespaces.json)
+
+### Location in archive
+- `config/netnamespaces.json`
+
+### Config ID
+`clusterconfig/netnamespaces`
 
-The Kubernetes api https://github.com/openshift/client-go/blob/master/network/clientset/versioned/typed/network/v1/netnamespace.go
-Response is an array of netNamespaces. Netnamespace contains Name, EgressIPs and NetID attributes.
+### Released version
+- 4.7.0
 
-* Location in archive: config/netnamespaces
-* Id in config: clusterconfig/netnamespaces
-* Since versions:
-  - 4.6.20+
-  - 4.7+
+### Backported versions
+- 4.6.20+
+
+### Changes
+None
 
 
 ## NodeLogs
 
-fetches control plane node logs from journal unit
+Collects control plane node logs from journal unit.
+
+### API Reference
+- https://docs.openshift.com/container-platform/4.9/rest_api/node_apis/node-core-v1.html#apiv1nodesnameproxypath
+
+### Sample data
+- [docs/insights-archive-sample/config/nodes/logs](./insights-archive-sample/config/nodes/logs)
+
+### Location in archive
+- `config/nodes/logs/{hostname}.log`
 
-Response see https://docs.openshift.com/container-platform/4.9/rest_api/node_apis/node-core-v1.html#apiv1nodesnameproxypath
+### Config ID
+`clusterconfig/node_logs`
 
-* Location in archive: config/nodes/logs/{hostname}.log
-* See: docs/insights-archive-sample/config/nodes/logs
-* Id in config: clusterconfig/node_logs
-* Since versions:
-  - 4.10+
+### Released version
+- 4.10.0
+
+### Backported versions
+None
+
+### Changes
+None
 
 
 ## Nodes
 
-collects all Nodes.
+Collects all cluster nodes.
```

Comment:
`Node` or `Nodes` is a resource kind. Better to denote it so, imho. This document uses at least four different ways to denote resource kinds (see another comment of mine), so I struggle to recommend one.